Nodejs/Product/TestAdapter/ProjectTestDiscoverer.cs

```diff
@@ -51,21 +49,24 @@ private void DiscoverTestsCore(IEnumerable<string> sources, IDiscoveryContext di
                 env["MSBuildExtensionsPath32"] = Path.Combine(root, "MSBuild");
             }
 
+            var frameworkDiscoverer = new FrameworkDiscoverer();
```

Comment:
There's no need to instantiate if there are no testItems. I think this can go later on the between lines 125-130

Reply:
yep